pipit/trace.py

```diff
@@ -681,3 +681,43 @@ def calc_exc_time_in_bin(events):
         df.insert(0, "bin_end", edges[1:])
 
         return df
+
+    @staticmethod
+    def multirun_analysis(
+        traces, metric_column="Timestamp (ns)", groupby_column="Name"
+    ):
+        """
+        Arguments:
+        traces - list of pipit traces
+        metric_column - the column of the metric to be aggregated over
+        groupby_column - the column that will be grouped by before aggregation
+
+        Returns:
+        A Pandas DataFrame indexed by the number of processes in the traces, the
+        columns are the groups of the groupby_column, and the entries of the DataFrame
+        are the aggregated metrics corresponding to the respective trace and group
+        """
+
+        # for each trace, collect a flat profile
+        flat_profiles = []
+        for trace in traces:
+            trace.calc_exc_metrics([metric_column])
```

Comment:
this might not be required right?

Reply:
good point, I think it's not required if the metric_column is "Timestamp (ns)" because flat profile takes care of that, but otherwise we do need it here